### DISCUSSION TITLE
Image size

### DISCUSSION THREAD

**Discussion by Ivan-Chupin** [umputun/remark42#1588] [General]
```
.raw-content img {
    max-height: 300px;
    max-width: 100%;
}
```

Why limit the image size? Why is it equal to 300px? If I remove this restriction, then in my case everything is displayed perfectly.

**Comment by akellbl4** [comment]
Image can be really long and that's the case when we limited the size because it ocuppied whole viewport's height.

**Reply by Ivan-Chupin** [reply]
I tested an image (674x708) without **max-height: 300px;**
Everything was displayed perfectly.

**Reply by paskal** [reply]
Can you please test narrow but very long image? The size you've posted is almost square.

**Reply by paskal** [reply]
https://placekitten.com/200/1200 example with current styles:
<img width="750" alt="image" src="https://user-images.githubusercontent.com/712534/233200617-f5d8219f-2c3b-40dd-8a0b-1c2248214f63.png">
Without these styles:
<img width="679" alt="image" src="https://user-images.githubusercontent.com/712534/233200688-2934d177-5897-4324-9c7e-b640609ad6ec.png">
I think this can be resolved. We should keep this style in place.

**Reply by akellbl4** [reply]
@paskal I can't find the issue or the or but I remember that we fixed it very intentionally

**Reply by Ivan-Chupin** [reply]
@akellbl4 In the current version of Remark42, tests show that there is no need for "**max-height: 300px;**"

**Comment by Ivan-Chupin** [comment]
With **max-height: 300px;**

![shrinked](https://user-images.githubusercontent.com/83099323/233372828-8ebdae4a-f37c-4292-87f7-c47f24eedc5b.png)

Without **max-height: 300px;** everything renders fine. I don't see any point in shrinking the image vertically. In addition, the user does not have the opportunity to view it without scaling. You can limit the height to 4K resolution (2160px).

![full](https://user-images.githubusercontent.com/83099323/233372895-e6f46faf-c0a9-4283-b052-45b95fe9a3ed.png)

**Reply by paskal** [reply]
My viewport height is 795 pixels. The screenshot I gave is how much of the picture fits on my screen. I think comments should be small and tidy and shouldn't be able to hijack multiple screens of a user with a long read image (as Twitter threads are sometimes attached).
